### PR TITLE
PMS-47-created frontend connection for update api

### PR DESCRIPTION
**Pull request by jaysingh321**

### CODE REVIEW

**Review by PraveenH18 — APPROVED** [review]

**Review by siddharthp1995 — COMMENTED** [review]
Resolve the conflicts. 

@PraveenH18 Review again after Jay resolves conflicts

**Review by PraveenH18 — APPROVED** [review]

File: src/view/Pages/Patients/edit/EditPatient.js [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 import "./EditPatient.scss";
+import { useDispatch } from "react-redux";
+import { updatePatientsRequest } from "../../../store/patientActions";
 
 const EditPatient = ({ patient, onCloseEditModal }) => {
   const [editedPatient, setEditedPatient] = useState({
+    hospitalId: patient.hospitalId,
     fullName: patient.fullName,
     age: patient.age,
-    gender: patient.gender,
```
Comment (pallawir) [inline src/view/Pages/Patients/edit/EditPatient.js:9]: Why gender is still showing as removed param?

File: src/view/Pages/Patients/edit/EditPatient.js [hunk 2]
```diff
@@ -58,25 +62,7 @@ const EditPatient = ({ patient, onCloseEditModal }) => {
                       onChange={handleInputChange}
                     />
                   </label>
-                  <label>Gender</label>
```
Comment (pallawir) [inline src/view/Pages/Patients/edit/EditPatient.js:61]: Why gender is still showing as removed?

File: src/view/Pages/Patients/edit/EditPatient.js (hunk 1 above)
Comment (jaysingh321) [reply src/view/Pages/Patients/edit/EditPatient.js:9]: for gender field will handle that on pms-38 which is create and edit modal  after we get pull of pms-47. once gender wil add then update request will work.

File: src/view/Pages/Patients/edit/EditPatient.js (hunk 2 above)
Comment (jaysingh321) [reply src/view/Pages/Patients/edit/EditPatient.js:61]: for gender field will handle that on pms-38 which is create and edit modal  after we get pull of pms-47. once gender wil add then update request will work.

**Review by pallawir — APPROVED** [review]

**Pull request merged**